functions.php

```diff
@@ -0,0 +1,26 @@
+<?php
+function getDatabase(): PDO{
+    $db = new PDO('mysql:host=DB;dbname=collectorApp', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+function queryDatabase(PDO $db){
+    $query = $db->query('SELECT `name`, `species`, `tvshow`, `catchphase`, `image`  FROM `cartoonChars`;');
+    $cartoonArray = $query->fetchAll();
+    return $cartoonArray;
+}
+
+function loopCartoons(array $cartoonArray): string{
```

Comment:
again, not very symantic. what does the loop do? and why? what about `displayCartoons`